### DISCUSSION TITLE
Where do i get zip file for OnlyFans DataScraper v7.6.1

### DISCUSSION THREAD

**Discussion by Fall0n** [UltimaHoarder/UltimaScraper#1836] [General]
OnlyFans DataScraper v7.4.1 can i only find on GitHub. Did have OnlyFans DataScraper v7.6.1 and my hard drive did die for me.. hope can get help

**Comment by SterlingSavariau** [comment]
https://github.com/DIGITALCRIMINALS/OnlyFans/releases/tag/v7.6.1
